src/pages/Home.tsx

```diff
@@ -43,23 +46,36 @@ export const Home: FC = () => {
             return;
         }
 
-        getEntityCustomerList(client, userId)
-            .then(async (customers: string[]) => {
-                const customerId = customers[0];
-
-                try {
-                    const { customer } = await getCustomer(client, customerId);
-                    const { orders } = customer;
+        // todo: change the getCustomer() method to accept an email
+        getCustomer(
```

Comment:
We can't find one customer by email. To find via email we need to use [Customers query](https://shopify.dev/api/admin-graphql/2021-10/queries/customers). This query may return more than one customer

Reply:
I think it would be do faster to Linking the customer one by one

Reply:
On LinkCustomer page, we just need to delete the existing association and add a new one

Reply:
Ok, that's fine